src/status_im/accounts/create/core.cljs

```diff
@@ -107,13 +107,14 @@
 
 (defn account-set-name [{{:accounts/keys [create] :as db} :db now :now :as cofx}]
   (fx/merge cofx
-            {:db                                  (assoc db :accounts/create {:show-welcome? true})
+            {:db                                              db
              :notifications/request-notifications-permissions nil
-             :dispatch                            [:navigate-to :home]}
+             :dispatch-n                                      [[:navigate-to :home]
```

Comment:
2 navigate?

Reply:
yes, we need to open home and immediately open modal welcome screen